lib/SILOptimizer/Mandatory/TFCanonicalizeCFG.cpp

```diff
@@ -118,6 +118,29 @@ void ConditionalSESERegion::print(llvm::raw_ostream &OS, unsigned indent) const
 // CFG Canonicalization Implementation
 //===----------------------------------------------------------------------===//
 
+// Our partitioning and other transformations can leave around lots of
+// unconditional branches between blocks that formerly had control edges.  Go
+// through and merge those to make later passes simpler.
+bool tf::contractUncondBranches(SILFunction *fn, DominanceInfo* DI, SILLoopInfo *LI) {
```

Comment:
will there be a caller where DI and LI are not NULL?

Reply:
Yes, I am preparing a PR that passes non-null for DI & LI.